modules/disable-automatic-upgrades.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * configuration/configure-automatic-upgrades.adoc
+:_module-type: PROCEDURE
+[id="disable-automatic-upgrades_{context}"]
+= Disabling automatic upgrades
+
+If you want to manage your secured cluster upgrades manually, you can disable automatic upgrades.
+
+.Procedure
+
+. In the RHACS portal, navigate to *Platform Configuration* → *Clusters*.
```

Comment:
For here and a few other places, can you change the arrow icon to "->"? (It will render as an arrow character when built)